examples/low_level_api/list-offers.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
```

Comment:
same here... these are _not_ modules... the name changes here are neither required nor in any way beneficial... and we risk breaking any links pointing to those examples from any external locations... please, let's not rename them unless we have a _very_ good reason